README.md

```diff
@@ -113,6 +113,32 @@ print(len(dh.collect_partitions(Path(__file__).parent / "src")))
 ```
 _[Dataset Usage Example](https://github.com/pycodehash/pycodehash/blob/main/example_data.py)_
 
+
+### Dependencies
+
```

Comment:
Would be good to also add a page in the docs and a link in the section `Detecting changes in data pipelines`